mlir/test/Integration/Dialect/Arith/CPU/multiplication.mlir

```diff
@@ -0,0 +1,98 @@
+// Tests mul operations and their variants (e.g. extended).
+// These tests are intended to be target agnostic: they should yield the same results 
+// regardless of the target platform.
+
+// RUN: mlir-opt %s --convert-scf-to-cf --convert-cf-to-llvm --convert-vector-to-llvm \
+// RUN:             --convert-func-to-llvm --convert-arith-to-llvm | \
+// RUN:   mlir-cpu-runner -e entry -entry-point-result=void \
+// RUN:                   --shared-libs=%mlir_c_runner_utils | \
+// RUN:   FileCheck %s --match-full-lines
+
+func.func @mulsi_extended_i1(%v1 : i1, %v2 : i1) -> (i1, i1) {
+  %low, %high = arith.mulsi_extended %v1, %v2 : i1
+  vector.print %low : i1
+  vector.print %high : i1
+  func.return %low, %high : i1, i1
+}
+
+func.func @mulsi_extended_on_i1() {
```

Comment:
You could add things like `vector.print "@mulsi_extended_on_i1"` and then add `CHECK-LABEL: mulsi_extended_on_i1`.